src/search_install.blp

```diff
@@ -1,80 +1,100 @@
 using Gtk 4.0;
 using Adw 1;
 
-template SearchInstallWindow : Adw.Window {
-    default-width: 500;
-    default-height: 450;
-    title: _("Install From Online");
-    modal: true;
+template $SearchInstallWindow: Adw.Window {
+  default-width: 500;
+  default-height: 450;
+  title: _("Install From Online");
+  modal: true;
 
-    Adw.ToolbarView main_toolbar_view {
-        [top]
-        HeaderBar header_bar {
-            show-title-buttons: false;
-            [start]
-            Button cancel_button {
-                label: _("Cancel");
-            }
-            [end]
-            Button install_button {
-                label: _("Install");
-                styles["suggested-action"]
-            }
+  Adw.ToolbarView main_toolbar_view {
+    [top]
+    HeaderBar header_bar {
+      show-title-buttons: false;
+
+      [start]
+      Button cancel_button {
+        label: _("Cancel");
+      }
+
+      [end]
+      Button install_button {
+        label: _("Install");
+
+        styles [
+          "suggested-action"
+        ]
+      }
+    }
+
+    [top]
+    Adw.Clamp {
+      Box {
+        margin-top: 6;
+        margin-start: 6;
+        margin-end: 6;
+
+        styles [
+          "linked"
+        ]
+
+        [start]
+        SearchEntry search_entry {
+          hexpand: true;
+          placeholder-text: _("Search for Flatpaks…");
         }
-        [top]
-        Adw.Clamp {
-            Box {
-                margin-top: 6;
-                margin-start: 6;
-                margin-end: 6;
-                styles["linked"]
-                [start]
-                SearchEntry search_entry {
-                    hexpand: true;
-                    placeholder-text: _("Search for Flatpaks, press Enter to search.");
-                }
-                [start]
-                MenuButton remotes_dropdown {
-                    label: _("All Remotes");
-                }
-            }
+
+        [start]
+        MenuButton remotes_dropdown {
```

Comment:
It will probably be removed, I'm not very happy with the structure of the Search Install window myself. So that whole window is most likely going to be reworked.